.github/workflows/scripts-setuptools-head.yml

```diff
@@ -11,9 +11,8 @@ jobs:
 
     runs-on: ubuntu-latest
     strategy:
-      max-parallel: 4
       matrix:
-        python-version: ['3.8', '3.9', '3.10', '3.11']
+        python-version: ['3.8', '3.9', '3.10', '3.11', '3.12', '3.13.0-rc.3']
```

Comment:
You don't need to specify the RC level. Just specifying "3.13" will get any latest available version, be it alpha, beta, RC or final.

Reply:
I did it like this because with 3.13 it did not work for me.

Reply:
We made the change in https://github.com/zopefoundation/meta/pull/239 but I forgot one detail, you also have to add `allow-prereleases: true` as an argument o the `setup-python` GHA action, which is safe to do because it only picks up prereleases if there is no final release.

Reply:
Thanks.  I have updated it to use `allow-prereleases: true`.

Reply:
3.13.0 final is released now at least in a couple of days it should be usable in GHA.